BXMSDOC-1331: Domain Exlporer filters docs (#2736)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/release-notes/0.14.0/ref-kogito-rn-new-features.adoc

== {PRODUCT} tooling

> === New Filter options on Management console and Improved Manage columns

Reviewer: ```suggestion
=== New filter options on {PRODUCT} Management Console and improved manage columns
```

Author: done

> Now Management console has new feature on Domain Explorer page, that allows you to filter domain data based on sections from the dropdowns. 

Reviewer: ```suggestion
In this release, a new feature on Domain Explorer page was added that allows you to filter domain related data based on attributes from a given domain.
```

Author: @cristianonicolai done. thanks

> There are three dropdowns allows you to select Field, Operator and Value based on which the data on the table will be filtered.

Reviewer: ```suggestion
There are three dropdowns that allow you to select an attribute, operator and the value on which the data on the domain data will be filtered.
```

could you also explain that operators and values are related to the type of attribute selected?
I think is also worth mentioning that by default doamin will now be filtered by active process instances.

Author: done
